### PR TITLE
slightly fixes up the solar class

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
The solar class is quite old at this point, and has a lot of small issues that could use fixing. This pr fixes some of these issues

This pr also removes the protolathe, in favour of a sci lathe!

## Why It's Good For The Game
fixes = good.

## Changelog

:cl:
fix: misc solar class fixes and changes ((((on the independent_solar))))
/:cl:

### CODE REVIEW

**Comment by Erikafox** [comment]
>fix: fixes wallmounts

on what ship

>fix: fixes wall lockers

which ship

>del: removes mounts outside the ship

what ship
>del: removes catwalks outside the ship

what ship is this on
>fix: fixes the zoning on the ship

when you say ship I assume you're referring to something
>fix: extends the airlock by one

we have an airlock?
>fix: adds a clothing locker

Oh good
>fix: adds wooden linings to the dormitories

Dorms??
>fix: moves wallmounts around (to avoid lights and wallmounts on the same wall)

I assume this is touching the komodo
>del: removes wallpipes
>del: removes tinyfans

god I wish

>add: adds holofans
>add: adds thruster blast doors

We already HAD those

**Comment by Erikafox** [comment]
tldr fix your changelog

**Comment by spockye** [comment]
> > fix: fixes wallmounts
> 
> on what ship
> 
> > fix: fixes wall lockers
> 
> which ship
> 
> > del: removes mounts outside the ship
> 
> what ship
> 
> > del: removes catwalks outside the ship
> 
> what ship is this on
> 
> > fix: fixes the zoning on the ship
> 
> when you say ship I assume you're referring to something
> 
> > fix: extends the airlock by one
> 
> we have an airlock?
> 
> > fix: adds a clothing locker
> 
> Oh good
> 
> > fix: adds wooden linings to the dormitories
> 
> Dorms??
> 
> > fix: moves wallmounts around (to avoid lights and wallmounts on the same wall)
> 
> I assume this is touching the komodo
> 
> > del: removes wallpipes
> > del: removes tinyfans
> 
> god I wish
> 
> > add: adds holofans
> > add: adds thruster blast doors
> 
> We already HAD those

on ALL of them
nerd

**Review by Erikafox — CHANGES_REQUESTED** [review]
you don't need more changelog than we make in an entire active week for assorted minor fixes on a ship

**Comment by goober3** [comment]
cool rework (of the independent_solar)

**Review by Erikafox — CHANGES_REQUESTED** [review]
please include an exterior docking port in this patch

**Comment by Erikafox** [comment]
image

**Comment by spockye** [comment]
what

**Comment by Erikafox** [comment]
I think we're covering most of these issues in anywalls. So I'm gonna close this. If there are more changes to be made after that is merged, make a new PR
